### DISCUSSION TITLE
Hack Club - Rejected for GitHub Education Benefits

### DISCUSSION THREAD

**Discussion by giragon6** [community/community#148209] [GitHub Education]
### Hi everyone,

Hack Club offers a program to get GitHub Education benefits by setting them as your school then providing proof of enrollment at your actual high school. After submitting my transcript at my actual high school for proof with my actual high school's name clearly visible, I was rejected for the following reason: 

"The image you selected does not appear to contain your school name. Your complete school name must appear in your document, not only the school logo. You may include multiple documents together. If your official document is not in English then you may photograph the original next to an English translation."

Is this because the document doesn't have the words "Hack Club" on it, even though the program should allow proof from your actual high school?

**Comment by samus-aran** [comment]
Hey @giragon6 

We’re happy to see that you’ve been successfully verified! 🎉 Welcome! Now, before you rush off to explore all your GitHub Education academic benefits, share with us what you’re most excited for in your upcoming studying for the year 2025! 🎆 

To help other Hack Club included in the current offer and having difficulties we do require a legible photo proof showing that you are a current Hack Club member. 👍

**Reply by bhatrichit10-ux** [reply]
what should be posted to show that you are a legible hack club member?
